src/agents/supervisor/agent.py

```diff
@@ -130,9 +137,15 @@ def _create_planner_chain(self, model: IModel) -> RunnableSequence:
 
     def _invoke_planner(self, state: SupervisorState) -> AIMessage:
         """Invoke the planner."""
+
+        messages = filter_messages_via_checks(
```

Comment:
are we not ignoring such following messages. I found them in the Langfuse traces. 
The information about resource is quite important.  I don't why it is not shown as system messages. 
Can you check whether if your langfuse traces still have them?
```
  AIMessage(
      content="The user query is related to: {'resource_api_version': 'gateway.kyma-project.io/v1beta1', 'resource_namespace': 'kyma-app-apirule-broken'}"
  ),
```

Reply:
we have system messages with the same content
```
SystemMessage(content="The user query is related to: {'resource_api_version': 'v1', 'resource_namespace': 'nginx-wrong-image'}", id='454913b7-4594-421d-828f-211eb841bfa4')
```